### DISCUSSION TITLE
Strange behaviour on top layer Orca Slicer 2.0

### DISCUSSION THREAD

**Discussion by Sapphire-Pro** [OrcaSlicer/OrcaSlicer#4864] [Dev]
Hi Everyone,

I got a strange Orca 2.0 behavour on several top layers on this stl
https://www.thingiverse.com/thing:2656594
I tried to slice it in both Arachne and Classic mode (this latter with Detect Thin Walls enabled to be able to get the writing on the top layer)
I tried to turn on and off every single option in standard profile.
It looks like Orca thinks that top layer is not flat.
I tried to slice the same stl in Prusa Slicer, Bambu Studio, Flash Print and Cura and all of these ones works as they should.

With Orca Slicer 2.0 :
![image](https://github.com/SoftFever/OrcaSlicer/assets/166008856/31528d7d-45f3-4642-ba80-b59bd8228464)

The same stl sliced with Bambu Studio 1.8.4.5.1
![image](https://github.com/SoftFever/OrcaSlicer/assets/166008856/b5e4c697-e59b-4ad3-8f40-4bc7c9f0b7eb)

What could it be ?

Thank you

**Comment by zoncky** [comment]
Same here. In Classic mode I have to turn on "Detect thin walls" for "Bridging test" to be sliced, but still there are those weird perimeters underneath. In Arachne no need for "Detect thin walls" (it's disabled in Arachne) but result is the same : weird perimeters underneath. 
All other slicers, ie. Prusa or Bambu, work correctly.

**Comment by dmazzer** [comment]
Disabling "Only one wall on top surfaces" solved this issue for me. But it doesn't make any sense....
